sdk/typescript/test/e2e/raw-signer.test.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
moved this test to e2e since it needs to get api version to decide which signing function to use